odoo/tools/i18n.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+from babel import lists
+from typing import Literal, Optional, Sequence, TYPE_CHECKING
+
+from odoo.tools.misc import babel_locale_parse, get_lang
+
+if TYPE_CHECKING:
+    import odoo.api
+
+
+def format_list(
+    env: odoo.api.Environment,
+    lst: Sequence[str],
```

Comment:
Wouldn't it be better to have a name like `items` or something to make it more explicit than `lst`?